### ISSUE TITLE
Got compile error on the client side

### ISSUE THREAD

**Issue by kimsama** [SaladbowlCreative/TicTacToe#7]
```
Assets/UnityPackages/CodeAnalysisVsHook/Editor/CodeAnalysisVsHook.cs(8,7): error CS0246: The type or namespace name `SyntaxTree' could not be found. Are you missing a using directive or an assembly reference?
```

Got the error on the client side with Unity 5.3.6f1. Though removing _CodeAnalysisVsHook.cs_ file runs ok without any problem.

**Comment by veblush** [comment]
Good catch! This error might be caused by the absence of VSTU(Visual Studio Tools for Unity).
If your dev environment is not Windows and VSTU, it makes sense but let me know if not.
Removing CodeAnalysisVsHook.cs is totally fine and I'll figure out a graceful fallback to handle missing VSTU.

**Comment by kimsama** [comment]
Though it was run on windows machine, I have not used VSTU since its version of 5.3.x.  

Thank you.
